.github/actions/deploy-environment/action.yml

```diff
@@ -47,15 +47,48 @@ runs:
       run: |
```

Comment:
line 21 should be value: ${{ steps.set_outputs.outputs.EXTERNAL_ACCESS_URL }} as we want to show the external url on the gh summary page